eng/Versions.props

```diff
@@ -5,6 +5,7 @@
     <SemanticVersioningV1>True</SemanticVersioningV1>
     <IncludeSourceRevisionInInformationalVersion>False</IncludeSourceRevisionInInformationalVersion>
     <UsingToolNetFrameworkReferenceAssemblies>True</UsingToolNetFrameworkReferenceAssemblies>
+    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-preview.1</MicrosoftNetFrameworkReferenceAssembliesVersion>
```

Comment:
Looks like this version still doesn't have the fix for dsplaisted/ReferenceAssemblyPackages#2

cc @dsplaisted

Reply:
cc @NikolaMilosavljevic

Reply:
We have this tracked at https://github.com/dotnet/core-sdk/issues/1912.  It should be fixed soon.

Reply:
Thanks!

Reply:
We just published an update that should fix this: https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/1.0.0-preview.2